test/noteListTest.js

```diff
@@ -16,5 +16,5 @@ it('Notelist returns latest note', function() {
   var noteList = new NoteList(note2);
   noteList.addNewNote("Hello");
   noteList.addNewNote("Goodbye");
-  return assert.equals(noteList.latestNote(), 'Goodbye');
+  return assert.equals(noteList.latestNote(), 'hello');
```

Comment:
Why have you changed this to hello and not goodbye

Reply:
# thoughtS?

Reply:
I've made a test fail so we can see if the printout of a failing test is working. 

We had an issue that all our tests are passing, because we're too good.

Reply:
Ok, this has really cleared this up for me.

Reply:
Super  interesting approach 
Think i'll go research it
🔢